<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="x_input">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="y_input">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_coord_btn">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="coords_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>202</width>
      <height>461</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>49</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="paint_lt"/>
   </widget>
   <widget class="QPushButton" name="count_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_le">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>510</y>
      <width>361</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
